templates/dynamic/includes/params/inputs/elastic.html

```diff
@@ -7,8 +7,7 @@
 
 {% block content %}
 <div class="inputs-block-header">
-  <h1>Macroeconomic Parameter</h1>
-  {% flatblock "taxbrain_dynamic_blurb" %}
+  <h1>Macroeconomic Parameters</h1>
```

Comment:
@GoFroggyRun And here's where the "s" got added back on.

Reply:
@hdoupe right. Git automatically takes care of some conflicts. That's why changes got reverted.